scripts/run-e2e-test.sh

```diff
@@ -22,37 +18,18 @@ function toggle_windows_scheduling(){
   done
 }
 
-if [[ -z "${OS_OVERRIDE}" ]]; then
-  OS_OVERRIDE=linux
-fi
-
-GET_CLUSTER_INFO_CMD="aws eks describe-cluster --name $CLUSTER_NAME --region $REGION"
-
-if [[ -z "${ENDPOINT}" ]]; then
-  CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD)
-else
-  CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD --endpoint $ENDPOINT)
-fi
-
 echo "Cordon off windows nodes"
 toggle_windows_scheduling "cordon"
 
-VPC_ID=$(echo $CLUSTER_INFO | jq -r '.cluster.resourcesVpcConfig.vpcId')
-ACCOUNT_ID=$(aws sts get-caller-identity | jq -r '.Account')
-
-echo "VPC ID: $VPC_ID"
-
 eksctl utils associate-iam-oidc-provider \
     --region $REGION \
     --cluster $CLUSTER_NAME \
     --approve
 
 echo "Creating AWSLoadbalancerController IAM Policy"
-curl -o iam-policy.json https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.2.1/docs/install/iam_policy.json
-
 aws iam create-policy \
     --policy-name AWSLoadBalancerControllerIAMPolicy \
```

Comment:
AWSLoadBalancerControllerIAMPolicy better to a name random generated with buildID/testID, such as `AWSLoadBalancerControllerIAMPolicy-$<something>`, so tests can be run concurrently